### DISCUSSION TITLE
VCS issue

### DISCUSSION THREAD

**Discussion by Almaszia** [oss-review-toolkit/ort#7756] [Q&A]
Hi all

I'm trying to understand why, whenever I use a Maven project for analysis, it consistently produces an error stating, "it lacks VCS information" and fails to generate any output.

**Comment by sschuberth** [comment]
Could you please copy & paste the exact error message that you see? And eventually run the analyzer with `--info` and `--stacktrace` to see more potentially helpful output.

**Reply by Almaszia** [reply]
I am new to this tool. I got this after it start maven analysis
 "INFO org.ossreviewtoolkit.analyzer.managers.utils.MavenSupport - Maven SCM connection '[git@github.com](mailto:git@github.com):ceki/slf4j.git' of project org.slf4j:slf4j-api:jar:1.7.5 lacks the required 'scm' prefix." followed by this "The VcsInfo(type=Git, url=https://github.com/kstyrc/embedded-redis.git, revision=embedded-redis-0.5, path=) of project 'Maven:com.github.kstyrc:embedded-redis:0.7-SNAPSHOT' cannot be found in Repository(vcs=VcsInfo(type=, url=, revision=, path=), vcsProcessed=VcsInfo(type=, url=, revision=, path=), nestedRepositories={}, config=RepositoryConfiguration(analyzer=null, excludes=Excludes(paths=[], scopes=[]), resolutions=Resolutions(issues=[], ruleViolations=[], vulnerabilities=[]), curations=Curations(packages=[], licenseFindings=[]), packageConfigurations=[], licenseChoices=LicenseChoices(repositoryLicenseChoices=[], packageLicenseChoices=[]))).
at org.ossreviewtoolkit.model.OrtResult.getFilePathRelativeToAnalyzerRoot(OrtResult.kt:222)
at org.ossreviewtoolkit.model.OrtResult.getDefinitionFilePathRelativeToAnalyzerRoot(OrtResult.kt:188)
at org.ossreviewtoolkit.model.config.Excludes.findPathExcludes(Excludes.kt:61)
at org.ossreviewtoolkit.model.OrtResult$projects$2.invoke(OrtResult.kt:153)
at org.ossreviewtoolkit.model.OrtResult$projects$2.invoke(OrtResult.kt:149)
at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)"

It is not creating any output. It is working for those projects that do not have any scm tag in the pom file.
